src/bench/nanobench.h

```diff
@@ -1905,7 +1978,7 @@ PerformanceCounters& performanceCounters() {
 // Windows version of doNotOptimizeAway
 // see https://github.com/google/benchmark/blob/master/include/benchmark/benchmark.h#L307
 // see https://github.com/facebook/folly/blob/master/folly/Benchmark.h#L280
-// see https://learn.microsoft.com/en-us/cpp/preprocessor/optimize
+// see https://docs.microsoft.com/en-us/cpp/preprocessor/optimize
```

Comment:
nit: Maybe backport a change from f84e445dee5f4c5d65cb702958701ff7b38ebdce into the original [repo](https://github.com/martinus/nanobench)?

Reply:
thanks, I'll backport this. Both URLs work currently though, so not much is lost.